### PR TITLE
Config page updates and fix for #65

### PR DESCRIPTION
**Pull request by GMart**
I've added updated descriptions and options for module config pages and regular config pages. Also spelling corrections in those pages.

### CODE REVIEW

File: src/pages/Peers.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export const PeersPage = (): JSX.Element => {
                           connection.getMetadata({ nodeNum: node.data.num }),
                           {
                             loading: "Requesting Metadata...",
-                            success: "Recieved Metadata",
```
Comment (sachaw) [inline src/pages/Peers.tsx]: Could you please revert this change, I have some rework to this page uncommitted,  I'll fix it there

**Review by sachaw — CHANGES_REQUESTED** [review]
Looks great, thanks!

File: src/pages/Peers.tsx (hunk 1 above)
Comment (GMart) [reply src/pages/Peers.tsx]: OK, reverted. I think you could have done a merge?

**Review by GMart — COMMENTED** [review]
Good to go for merge

**Pull request merged**